install/installer/trafodion_config_default

```diff
@@ -125,6 +125,9 @@ export PASSWORD="admin"
 # hadoop cluster name
 export CLUSTER_NAME=""
 
+#Trafodion Cluster name, normally the same as Cloudera cluster name
+export CLUSTERNAME=""
```

Comment:
Glad I saw this comment otherwise I'd have no clue about the difference between CLUSTER_NAME and CLUSTERNAME. Maybe these should be named differently? (Perhaps TRAFODION_CLUSTER_NAME for one of them?)